packages/animations/lib/src/shared_axis_transition.dart

```diff
@@ -487,7 +504,7 @@ class _ExitTransition extends StatelessWidget {
         return FadeTransition(
           opacity: _fadeOutTransition.animate(animation),
           child: Container(
-            color: Theme.of(context).canvasColor,
+            color: fillColor ?? Theme.of(context).canvasColor,
```

Comment:
Why did we have to give this a background color at all? Couldn't it just be unopinionated and let whatever color is behind it shine through?

Reply:
I think this was an oversight. Would it be better to just remove the color parameter altogether?

Reply:
I think so. It seems the transition shouldn't really have an opinion about the background color.

Reply:
I actually remember why now. This becomes a problem if you use `SharedAxisPageTransitionsBuilder` without a `Container` with some sort of fill color. What ends up happening is while both the transition is occurring, you'll get a black background if there's nothing behind those routes since both while become transparent. This isn't a problem when using the PageTransitionSwitcher because it does have a background color behind the coming and going widgets.

Reply:
Ah, that makes sense.